oaknational/Oak-Web-Application
# fix: policy page links styling [LESQ-556]

## Description

Music year: 1987

- Fix external and internal link styling (blue underline) on policy pages

## Issue(s)

Fixes LESQ-556

## How to test

1. Go to [https://deploy-preview-3008--oak-web-application.netlify.thenational.academy/legal/privacy-policy](https://deploy-preview-3008--oak-web-application.netlify.thenational.academy/legal/privacy-policy) or any other policy page
2. You should see links which are correctly tagged in Sanity are blue and underlined

## Screenshots

How it used to look (delete if n/a):
<img width="660" alt="image" src="https://github.com/user-attachments/assets/28eb6472-33c5-496d-a367-917ecfcf4cf4">

How it should now look:
<img width="630" alt="image" src="https://github.com/user-attachments/assets/ede6599c-77dd-4922-b8b1-9f12e1f62160">

## Checklist
